### DISCUSSION TITLE
Input/Output error

### DISCUSSION THREAD

**Discussion by MaelAbgrall** [trapexit/mergerfs#930] [Q&A] [answered]
Hello,

I'm trying to find an issue I have that made my disks throw an IO error...
![img](https://i.imgur.com/MzYBC3u.png)

Following [some advice on the unix stackexchange](https://unix.stackexchange.com/a/39908/282565), I ran the dmesg command and saw those lines that seems to point to mergerfs:

![img](https://i.imgur.com/WyklGvB.png)

Any idea what is going on?

--- 
If it can help, the issue first happened while watching a movie with Jellyfin on docker, the streaming stopped in the middle. Quickly some other services on docker started throwing errors (often SQLite being in read-only mode).

**Comment by trapexit** [comment, marked as answer]
As the error says there is an error reading the sector/block. Your drive has bad blocks.

**Reply by MaelAbgrall** [reply]
Thanks for the info, I restarted the server and it's working fine so far. 
I ran ```smartctl -t long /dev/sdX``` all night on the two disks, and they both passed the health self assessment?

**Reply by trapexit** [reply]
Could be a number of other things then. Perhaps the interconnects acting up. I've had bad enclosures which made it look like the drives in them were bad.

**Reply by MaelAbgrall** [reply]
all right thanks!
and yes, they are on an external bay of okayish quality

**Reply by trapexit** [reply]
If you see this kind of thing happen again you might want to consider replacing it. At least perhaps reordering the drives to see if a specific port is bad. That happened to me with an esata enclosure years ago. 3 of the ports were fine. The 4th would randomly claim the drive had errors no matter the drive I put in it.
